src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -14,24 +14,32 @@
 
 .heatmap {
   flex: none;
+  box-sizing: content-box;
+}
+
+/* R3F's root element */
+.canvasWrapper {
+  overflow: visible !important; /* show axis */
```

Comment:
The `canvasWrapper` no longer includes the size of the axis, so we have to make overflow visible. This has a side-effect which requires setting `pointer-events: none` below.

Reply:
I've reviewed the comment here.